code/datums/outfits/event/outfit_events.dm

```diff
@@ -100,4 +100,60 @@
 	belt_contents = list(
 		/obj/item/device/multitool = 1
 	)
-	id_access = "Lance Operative"
\ No newline at end of file
+	id_access = "Lance Operative"
+
+/datum/outfit/admin/event/sol_marine
+	name = "Solarian Marine"
+
+	uniform = /obj/item/clothing/under/rank/fatigues
+	shoes = /obj/item/clothing/shoes/magboots
+	gloves = /obj/item/clothing/gloves/combat
+	l_ear = /obj/item/device/radio/headset/syndicate
+	glasses = /obj/item/clothing/glasses/thermal
+	l_pocket = /obj/item/tank/emergency_oxygen/double
+	r_pocket = /obj/item/crowbar/red
+	id = /obj/item/card/id/syndicate
+	head = /obj/item/clothing/head/helmet/space/void/sol
+	suit = /obj/item/clothing/suit/space/void/sol
+	mask = /obj/item/clothing/mask/gas/tactical
+	back = /obj/item/tank/jetpack/carbondioxide
+	suit_store = /obj/item/gun/projectile/automatic/rifle/sol
+
+	belt = /obj/item/storage/belt/military
+	belt_contents = list(
+			/obj/item/ammo_magazine/c762/sol = 3,
+			/obj/item/ammo_magazine/mc9mm = 2,
+			/obj/item/shield/energy = 1,
+			/obj/item/melee/energy/sword = 1,
+			/obj/item/grenade/frag = 1,
+			/obj/item/grenade/flashbang = 1
+	)
+
+
+	accessory = /obj/item/clothing/accessory/holster/hip/brown
+	accessory_contents = list(/obj/item/gun/projectile/pistol/sol = 1)
+
+/datum/outfit/admin/event/sol_marine/get_id_access()
+	return get_all_station_access()
```

Comment:
Do sol marines need to start with full station access? Considering they're an external enemy force I think it'd make sense for them not to. Depends on how they will be used.

Reply:
Ferner has got a pretty good point.
Adding new access is as simple as spawning in the captains spare for them to swipe.

But removing it is more difficult.

Reply:
Handled.